src/reactor/mod.rs

```diff
@@ -469,7 +469,8 @@ impl Inner {
                               mio::Token(TOKEN_START + entry.index() * 2),
                               mio::Ready::readable() |
                                 mio::Ready::writable() |
-                                platform::hup(),
+                                platform::hup() |
+                                platform::aio(),
```

Comment:
Oh this is something I should have fixed earlier, mind changing this to `platform::all()` and folding both `hup` and `aio` into that one function?